dist/trackomatic.js

```diff
@@ -0,0 +1,904 @@
+	    return (config.REGEX.COOKIE.exec(document.cookie) || []).pop();
+	  },
+
+	  /**
+	   * Writes cookies. This function is a no-op if `_trackomatic.options.nocookie` is `true`.
+	   * @param {string} name - the name for the cookie value
+	   * @param {mixed} value - the value to be stored as a cookie
+	   * @param {number} days - the number of days to store the cookie for
+	   */
+	  createCookie: function createCookie(name, value, days) {
+	    if (_trackomatic.options.nocookie) return;
+
+	    var expires = days ? ' expires=' + new Date(days * 864E5 + new Date().getTime()).toGMTString() : '';
+	    document.cookie = name + '=' + value + expires + '; path=/; domain=' + document.location.hostname;
+	  },
+
+	  /**
+	   * Debounces function calls.
+	   */
+	  debounce: function debounce(func, wait, immediate) {
+	    var timeout;
+
+	    return function () {
+	      var context = this;
+	      var args = arguments;
+
+	      var later = function later() {
+	        timeout = null;
+	        if (!immediate) func.apply(context, args);
+	      };
+
+	      var callNow = immediate && !timeout;
+
+	      clearTimeout(timeout);
+	      timeout = setTimeout(later, wait);
+
+	      if (callNow) func.apply(context, args);
+	    };
+	  },
+
+	  /**
+	   * Gets the viewport size.
+	   */
+	  getViewportSize: function getViewportSize() {
+	    if (typeof window.innerWidth !== 'undefined') {
+	      return {
+	        width: window.innerWidth,
+	        height: window.innerHeight
+	      };
+	    } else if (typeof document.documentElement !== 'undefined' && typeof document.documentElement.clientWidth !== 'undefined' && document.documentElement.clientWidth !== 0) {
+	      return {
+	        width: document.documentElement.clientWidth,
+	        height: document.documentElement.clientHeight
+	      };
+	    } else {
+	      return {
+	        width: config.BODY.clientWidth,
+	        height: config.BODY.clientHeight
+	      };
+	    }
+	  },
+
+	  /**
+	   * Rounds numbers.
+	   */
+	  roundXtoY: function roundXtoY(x, y) {
+	    var upper = Math.ceil(x / y) * y;
+	    var lower = Math.floor(x / y) * y;
+
+	    return x - lower < upper - x ? lower : upper;
+	  },
+
+	  createNavigationHandler: function createNavigationHandler(url) {
+	    return function () {
+	      window.location = url;
+	    };
+	  },
+
+	  getHost: function getHost(url) {
+	    return url.replace(config.REGEX.HTTP_PROTOCOL, '').split('/')[0];
+	  },
+
+	  getLink: function getLink(node) {
+	    if (node === document.documentElement) {
+	      return false;
+	    } else if (node.tagName.toLowerCase() === 'a') {
+	      return node;
+	    } else {
+	      return util.getLink(node.parentNode);
+	    }
+	  },
+
+	  getURLParams: function getURLParams() {
+	    var map = {};
+	    location.href.replace(config.REGEX.QUERY_PARAMS, function (m, k, v) {
+	      return map[k] = v;
+	    });
+	    return map;
+	  }
+	};
+
+	module.exports = util;
+
+/***/ }
+/******/ ]);
```

Comment:
Sometimes i wonder if `trackomatic` should be the minified file and `trackomatic.debug` should be the unminified one, with extra console.logs and etc. It seems like this is a file we would want to make 100% clear than no one should use in production.